receiver/podmanreceiver/README.md

```diff
@@ -1,12 +1,15 @@
 # Podman Stats Receiver
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
```

Comment:
Since this component does not have a code owner, I think we should mark it as alpha.